source/plugins/ruby/filter_cadvisor2mdm.rb

```diff
@@ -345,6 +379,7 @@ def ensure_cpu_memory_capacity_set
         end
       elsif @@controller_type.downcase == "daemonset"
         capacity_from_kubelet = KubeletUtils.get_node_capacity
+        allocatable_from_kubelet = KubeletUtils.get_node_allocatable(@@isWindows)
```

Comment:
two questions 1) should  be winNode instead @@isWindows here 2) i see that, you are passing windows to the allocatable but not capacity. this will be an issue on windows nodes, right?

Reply:
I checked the windows container. I see that we actually have a node ip environment variable defined for windows containers too and the get_node_capacity function is also using it. ([linked here](https://github.com/microsoft/Docker-Provider/blob/ci_dev/source/plugins/ruby/CAdvisorMetricsAPIClient.rb#L106))

I'm updating my code to use this too now similar to capacity.